applications/IGAStructuralMechanicsApplication/custom_elements/meshless_shell_element.cpp

```diff
@@ -1108,34 +820,37 @@ void MeshlessShellElement::CalculateAll(
 	double thickness = this->GetProperties().GetValue(THICKNESS);
 	DCurvature = DMembrane*(pow(thickness, 2) / 12);
 
-	//Deformations for Non-linear force Vector
-	Vector StrainDeformation;
-	Vector CurvatureDeformation;
+	//Local Cartesian Foreces and Moments
+	Vector ForceVector_in_Q_coordinates = ZeroVector(3);
```

Comment:
you don't need to first initialize this to zero. i mean it would be shorter and run faster to do 

Vector ForceVector_in_Q_coordinates = prod(trans(DMembrane), StrainVector_in_Q_coordinates);